packages/@ourworldindata/grapher/src/axis/AxisViews.tsx

```diff
@@ -242,52 +261,64 @@ export class VerticalAxisComponent extends React.Component<{
             showTickMarks,
         } = this.props
         const { tickLabels, labelTextWrap } = verticalAxis
-
-        const tickMarks = showTickMarks ? (
-            <VerticalAxisTickMarks
-                tickMarkYPositions={tickLabels.map((label) =>
-                    verticalAxis.place(label.value)
-                )}
-                tickMarkLeftPosition={bounds.left + verticalAxis.width}
-                color={SOLID_TICK_COLOR}
-            />
-        ) : undefined
+        const tickSize = 5
 
         return (
-            <g className="VerticalAxis">
+            <g
+                id={makeIdForHumanConsumption("vertical-axis")}
+                className="VerticalAxis"
+            >
                 {labelTextWrap &&
                     labelTextWrap.renderSVG(
                         -verticalAxis.rangeCenter - labelTextWrap.width / 2,
                         bounds.left,
                         {
+                            id: makeIdForHumanConsumption("axis-label"),
```

Comment:
I have renamed `axis-label`, but since we often have multiple Graphers on one page, I expect most of these IDs to be present on a page multiple times. I think as long as we don't use them, we should be fine. (I hope `makeIdForHumanConsumption` makes it clear that we shouldn't)